doc/release-notes/master.txt

```diff
@@ -10,9 +10,10 @@ Updates since v1.1.0
 
 Vital
 
- * Removed the VITAL_FOREACH macro and vital_foreach.h.  This was left over
```

Comment:
Please don't remove specific mention of VITAL_FOREACH.  I know it falls under the broader category of removing C++11 compatibility shims, but it was the most widely used shim, both in KWIVER and in MAP-Tk, and had its own header file, which is now removed.  This is a big change that users of KWIVER likely need to adjust their code to compensate for.  I would like the release notes to specifically call out that the VITAL_FOREACH macro and header are now gone.

Reply:
As an alternative, it would be kinder to the projects that use kwiver and vital_foreach.h to leave the header for the short term and maybe add some deprecated warning to allow our customers to phase out the macro.

Reply:
That VITAL_FOREACH branch already landed.  However, I would not be against putting that header back with some sort of deprecation message, at least until the next release.  However, that should be a new branch, and I don't know the right way to handle deprecation notices in KWIVER.

Reply:
If we do that, in this instance we *may* be able to get away with just sticking a `#warning` in the header. I'm not sure that it is possible in general to mark a *macro* as deprecated. (Functions, yes. Other stuff...)

In fact, since the objective is to entirely remove the header, we would probably want to do that even if we *could* flag uses of the macro...

That said, I talked to Linus briefly, and I'm not sure we're convinced it's a big enough issue to warrant "un-removing" it...

Reply:
A deprecated warning would be nice (there are some solutions on StackOverflow), but if it is difficult, we do not have that many customers of KWIVER, so we could just work through it. If there are many external customers (not in Kitware), it would be nicer to have a softer failure.